zp-relayer/validation/tx/validateTx.ts

```diff
@@ -195,6 +188,17 @@ function checkMemoPrefix(memo: string, txType: TxType) {
   return new TxValidationError(`Memo prefix is incorrect: ${numItemsSuffix}`)
 }
 
+export async function checkWithdrawalTransfer(token: Contract, address: string, amount: BN) {
+  try {
+    await token.methods.transfer(address, amount.toString(10)).call({
```

Comment:
To avoid issues with unavailability of tokens on the pool contract due to compounding, I think we can check if a transfer of 0 wei succeeds. I think it'll perform the same blacklist checks for most tokens.

Reply:
Yes, I think should work fine. Updated